resources/views/edit.blade.php

```diff
@@ -4,7 +4,7 @@
 
     @include($form)
 
-    @if (count($errors) > 0)
+    @if (isset($errors) && count($errors) > 0)
```

Comment:
Interesting. I did not make that PR for fun - the error was happening on my box ;-)
Which Laravel & PHP version you are own? (I used the latest stable Laravel + PHP7)